ui/src/pages/repos/CreateRepoForm.tsx

```diff
@@ -21,7 +21,7 @@ interface form {
   button?: any;
 }
 
-export default function CreateRepoForm(props: form = {}) {
```

Comment:
The interface form is unused now. Remove it?

Reply:
Yeah, in the new line, I removed the type `form` and default value `{}` but left the `props` argument there for easy customization and control in the future. Good?

Reply:
Sounds good.